terraformprovider/resource_object_group.go

```diff
@@ -133,9 +151,36 @@ func resourceObjectGroup() *schema.Resource {
 				Default:  "",
 			},
 		},
+		CustomizeDiff: resourceObjectGroupCustomizeDiff,
 	}
 }
 
+func resourceObjectGroupCustomizeDiff(ctx context.Context, diff *schema.ResourceDiff, meta interface{}) error {
+	column_selection_list := diff.Get("column_selection").(*schema.Set)
+	if column_selection_list.Len() > 0 {
```

Comment:
If I correctly understand the next line, we don't really expect that this len will ever be bigger than 1? So it should be either 0 or 1?
If my assumption is correct, should we maybe return some fmt.error in case if the len is bigger than 1?